Issue 1965

Naaaaiix opened the pull request:
Fixes #1965

Naaaaiix committed:
Fixes #1965

Naaaaiix committed:
Fix tests

Naaaaiix committed:
Merge branch 'master' into issue-1965

Naaaaiix committed:
Fix tests disabled on background tests

javadsamiee reviewed (commented):
Excellent work 👍🏻  I think it's better to have a reset for "Colour Opacity"

elzadj reviewed (changes requested):
I think we need to hide the palette's opacity when we use a Custom color. 
![Edit-Page-‹-Local-Wordpress-—-WordPress (35)](https://user-images.githubusercontent.com/2401618/129321954-54cc9d3e-6de8-4db8-ad98-5e74f0411904.jpg)

Naaaaiix committed:
Palette opacity bar disappear with Custom Color
Custom Color keeps color and opacity when switching from Palette
Add reset button for Palette opacity

elzadj reviewed (changes requested):
Thanks @Naaaaiix 

There is a bug with the Opacity number - first I cannot change it with keyboard, and then it breaks with

```
react-dom.js?ver=16.13.1:82 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
```


https://user-images.githubusercontent.com/2401618/129337691-d78f0d8a-fbd4-40e9-904c-a5083d7cadaa.mp4


[blocks.txt](https://github.com/yeahcan/maxi-blocks/files/6981428/blocks.txt)

Naaaaiix committed:
Fix opacity palette manual change

Naaaaiix committed:
Merge branch 'master' into issue-1965

Naaaaiix committed:
Merge branch 'master' into issue-1965

Naaaaiix committed:
Fix box-shadow test

Naaaaiix committed:
Merge branch 'master' into issue-1965
